tests/test_constraints.py

```diff
@@ -99,3 +127,208 @@ def fast(self, auxdata, pars):
     slow_result = pyhf.tensorlib.tolist(slow(m, auxd, pars))
     fast_result = pyhf.tensorlib.tolist(fast(m, auxd, pars))
     assert pytest.approx(slow_result) == fast_result
+
+
+def test_ok(backend):
```

Comment:
Sorry, missed this earlier, but can we get a more descriptive test title? It is a few hundred lines, so having a more descriptive name will help people reading it in the future.

Reply:
good catch. done!

Reply:
thanks!